test/shared/utils/NestedJsonReadsSpec.scala

```diff
@@ -22,8 +22,7 @@ import shared.utils.NestedJsonReads._
 
 class NestedJsonReadsSpec extends UnitSpec {
 
-  val firstOutput: JsValue = Json.parse(
-    """{
+  val firstOutput: JsValue = Json.parse("""{
```

Comment:
Is it worth improving this old test? With these chunks of JSON so far from where they are used (sometimes only once) and with meaningless names it's tricky to see what's special about them. 

Do we even use this anymore?

Reply:
yep, it struck me as a lame old test as well.